site/src/components/radio.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Radio
+description: The radio component in the Atlas Design System
+template: standard
+figmaEmbed: https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FuVA2amRR71yJZ0GS6RI6zL%2F%25F0%259F%258C%259E-Atlas-Design-Library%3Fnode-id%3D838%253A1096"
+---
+
+# Radio
+
+A radio button allows users to select a single option. It is usually associated with form submissions.
+
+<div class="table-wrapper margin-top-xs">
+	<table class="table">
+		<thead>
+			<tr>
+				<th>Type</th>
+				<th>Default</th>
+				<th>Disabled</th>
+			</tr>
+		</thead>
+		<tbody>
+			<tr>
+				<td>Unchecked</td>
+				<td>
+					<label class="radio" title="">
+                    	<input
+                    		type="radio"
+                    		name="b16fe0d7-9dfa-460d-a8c4-8b39cad04240"
+                    		value="4"
+                            class="radio-dot"
+                    	/>
+                    	<span class="radio-label-text">50%</span>
+                    </label>
+				</td>
+				<td>
+					<label class="radio" title="">
+                    	<input
+                    		type="radio"
+                    		name="b16fe0d7-9dfa-460d-a8c4-8b39cad04240"
+                    		value="4"
+                            class="radio-dot"
+                            disabled
+                    	/>
+                    	<span class="radio-label-text">70%</span>
+                    </label>
+				</td>
+			</tr>
+			<tr>
+				<td>Checked</td>
+				<td>
+					<label class="radio" title="">
+                    	<input
+                    		type="radio"
+                    		name="b16fe0d7-9dfa-460d-a8c4-8b39cad04241"
+                    		value="4"
+                            class="radio-dot"
+                            checked
+                    	/>
+                    	<span class="radio-label-text">50%</span>
+                    </label>
+				</td>
+				<td>
+					<label class="radio" title="">
+                    	<input
+                    		type="radio"
+                    		name="b16fe0d7-9dfa-460d-a8c4-8b39cad04241"
+                    		value="4"
+                    		class="is-checked radio-dot"
+                            disabled
+                    	/>
+                    	<span class="radio-label-text">70%</span>
+                    </label>
+				</td>
+			</tr>
+		</tbody>
+	</table>
+</div>
+
+## Usage
+
+Here is an example of a group of radio buttons. You can group multiple ratio buttons together within a `fieldset` and `radios` container.
+Make sure to use the same value for the `name` attribute within a group of radio buttons.
+
+```html
+<fieldset class="field border-none">
```

Comment:
I think we should reduce the example here to include `radios` and `radio` only. Form patterns should be listed separately and will require further design guidance.

Reply:
Removed fieldset and legend